instance/tests/models/test_openedx_appserver.py

```diff
@@ -132,6 +133,30 @@ def test_provision_unhandled_exception(self, mocks):
         self.assertFalse(result)
         mocks.mock_provision_failed_email.assert_called_once_with("AppServer deploy failed: unhandled exception")
 
+    @patch('instance.models.openedx_appserver.get_username_list_from_team')
```

Comment:
Update the tests according to the feedback above (re: removing the `get_username_list_from_team` function)